modules/economy-system/events/messageCreate.js

```diff
@@ -0,0 +1,17 @@
+const {balance} = require('../economy-system');
+
+module.exports.run = async function (client, message) {
+    if (!client.botReadyAt) return;
+    if (!message.guild) return;
+    if (message.author.bot) return;
+    if (message.guild.id !== client.config.guildID) return;
+
+    const config = client.configurations['economy-system']['config'];
+
+    if (config['messageDrops'] === 0) return;
+    if (Math.floor(Math.random() * config['messageDrops']) !== 1) return;
+    const toAdd = Math.floor(Math.random() * (interaction.config['messageDropsMax'] - interaction.config['messageDropsMin'])) + interaction.config['messageDropsMin'];
```

Comment:
```suggestion
    const toAdd = Math.floor(Math.random() * (config['messageDropsMax'] - config['messageDropsMin'])) + config['messageDropsMin'];
```
interaction is not defined here